mitodl/micromasters
# allow link in table block

#### What are the relevant tickets?
Fixes #4235 

#### What's this PR do?
Previously we were rendering tables as simple text, this PR will enable HTML rendering inside the tables.

#### How should this be manually tested?

- Add program tab page.
- Insert HTML links inside the table cell.
- Preview table, links should render as HTMl.


#### Screenshots (if appropriate)
**Editing in table**

![screen shot 2019-01-22 at 2 39 42 pm](https://user-images.githubusercontent.com/4245618/51527973-7b3b9080-1e57-11e9-956b-03c969e7ca27.png)

**Rendering**

![screen shot 2019-01-22 at 2 36 35 pm](https://user-images.githubusercontent.com/4245618/51527972-7b3b9080-1e57-11e9-83ed-afd040e1d06f.png)
